index.js

```diff
@@ -98,10 +98,13 @@ function isTypedArray (value) {
   return typedArrayPrototypeGetSymbolToStringTag.call(value) !== undefined
 }
 
-function stringifyTypedArray (array, separator) {
+function stringifyTypedArray (array, separator, maximumBreadth) {
   if (array.length === 0) {
     return ''
   }
+  if (array.length > maximumBreadth) {
+    array = array.slice(0, maximumBreadth)
+  }
   const whitespace = separator === ',' ? '' : ' '
   let res = `"0":${whitespace}${array[0]}`
   for (let i = 1; i < array.length; i++) {
```

Comment:
This would copy the portion of the original typed array which is pretty slow. Instead, just define the maximum:

```suggestion
  if (array.length < maximumBreadth) {
    maximumBreadth = array.length
  }
  const whitespace = separator === ',' ? '' : ' '
  let res = `"0":${whitespace}${array[0]}`
  for (let i = 1; i < maximumBreadth; i++) {
```